### DISCUSSION TITLE
Operators and EGMME in tergms

### DISCUSSION THREAD

**Discussion by martinamorris** [statnet/tergm#63] [Ideas]
Now that we have more operators for tergm (`Cross() and Change()`), I'm trying to understand whether this has implications for the range of models that could be fit with EGMME.

1. What operators can be used in the EGMME context?
2. Can only separable models be used?

In particular, is it possible to specify a `Cross() + Change()` model?  It seems like this should be possible, by specifying

```
tergm(nw ~ Cross(~edges) + Change(~offset(edges)),
  targets = Cross() terms
  offset.coef = coef(s) for Change() terms
```

I know the machinery isn't currently there to do this, but would it be possible?

@krivit @sgoodreau

**Comment by krivit** [comment]
For EGMME, if the model and the targets aren't of just the right form, you have to specify initial parameters manually. Otherwise, it should work seamlessly.

In terms of nesting operators, some can go outside the temporal operators---those that don't mutate the network. For example, `Sum(~Form())` is OK. For others, you can use operators *inside* the temporal operators: e.g., `Form(~S())` is OK but `S(~Form())` will probably fail.
